mzLib/Proteomics/Fragmentation/Product.cs

```diff
@@ -32,67 +41,52 @@ public struct Product
             SecondaryFragmentNumber = secondaryFragmentNumber;
         }
 
-        public string Annotation
```

Comment:
can i assume this was unused?

Reply:
This is used quite often, but the implementation for how to parse the annotation from the object data has been moved to the interface. The new getter for Annotation (line 23) calls the interfaces implementation of this method. The same is true for the RNAProduct class

Reply:
will you show me this some time?